pkg/joinserver/joinserver.go

```diff
@@ -155,6 +156,20 @@ func validateCaller(dn pkix.Name, addr string) error {
 	return nil
 }
 
+// wrapKeyIfKEKExists wraps the given key with the KEK label.
+// If the configured key vault cannot find the KEK, the key is returned in the clear.
+func (js *JoinServer) wrapKeyIfKEKExists(key types.AES128Key, kekLabel string) (*ttnpb.KeyEnvelope, error) {
+	env, err := cryptoutil.WrapAES128Key(key, kekLabel, js.KeyVault)
+	if errors.IsNotFound(err) {
```

Comment:
This may do a whole bunch of type assertions, maybe only do it if `err != nil` ?